src/index.html

```diff
@@ -1,14 +1,58 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect"
+    href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet">
+    <link rel="stylesheet"
+    href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="link__home">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav>
+        <ul class="container">
+          <li class="nav__list">
+            <a href="#apple" class="nav__link is-active">apple</a>
+          </li>
+          <li class="nav__list">
+            <a href="#samsung" class="nav__link">samsung</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__list">
            <a href="#apple" class="nav__link is-active">apple</a>
          </li>
          
          <li class="nav__list">
            <a href="#samsung" class="nav__link">samsung</a>
          </li>
```
Add blank line between multiple sibling blocks of html